Source/WebCore/page/PointerCaptureController.cpp

```diff
@@ -313,6 +313,11 @@ void PointerCaptureController::dispatchEventForTouchAtIndex(EventTarget& target,
         dispatchEnterOrLeaveEvent(eventNames().pointerenterEvent, *currentTarget);
     }
 
+    if (pointerEvent->type() == eventNames().pointercancelEvent) {
+        cancelPointer(pointerEvent->pointerId(), platformTouchEvent.touchLocationAtIndex(index), pointerEvent.ptr());
```

Comment:
WPE will require something like `documentPointForWindowPoint(<frame>, platformTouchEvent.touchPoints()[index].pos())` I suppose

Reply:
In the interest of being able to merge this soon, I just added a platform guard for iOS family instead